### PR TITLE
Added support for travis CI

### PR DESCRIPTION
**Pull request by felipewmartins**
Added the travis.yml file, and created a test for udcli. #3

### CODE REVIEW

File: test/udcli-test.js [hunk 1]
```diff
@@ -7,16 +7,16 @@ const roi = require('roi');
...
   const options = {
     'endpoint': 'http://api.urbandictionary.com/v0/define?term=lgtm'
-  }
+  };
 
 test('Should be equal Looks Good To Me ', t => {
 	const options = {
     'endpoint': 'http://api.urbandictionary.com/v0/define?term=lgtm'
-  }
-  var resultJson = "An acronym for \"Looks Good To Me\", often used as a quick response after reviewing someone's essay, code, or design document"
```
Comment (helio-frota) [inline test/udcli-test.js]: we should use let or const instead var.

**Comment by helio-frota** [comment]
@felipewmartins thanks for the PR!

Can you squash the four commits into one ?

**Comment by felipewmartins** [comment]
Hi @helio-frota.
 I will adjust and join the commits.

Thanks

**Comment by helio-frota** [comment]
@felipewmartins the build is failing but I'm going to merge and take a look later.
Also feel free to try to solve if you interested.
Thanks!

**Comment by felipewmartins** [comment]
Thanks @helio-frota I will try to solve this problem.

**Pull request merged**